src/components/Header.js

```diff
@@ -2,10 +2,12 @@ import React from 'react'
 import "../css/main.css";
 import {Link} from "react-router-dom";
 import { useCart } from '../context/CartContext';
+import { useWishlist } from '../context/WishContext';
```

Comment:
create index file and export  from one file that will one line of code.
like 
import { useCart , useWishlist} from "../context"

Reply:
ok sure , i will try